privaterelay/templates/includes/dashboard_onboarding.html

```diff
@@ -2,28 +2,48 @@
 {% load ftl %}
 {% ftlconf bundle='privaterelay.ftl_bundles.main' %}
 
-
-<h2 class="dashboard-onboarding-header">{% ftlmsg 'onboarding-headline' %}</h2>
-<ol class="dashboard-onboarding-list">
-  <li class="dashboard-onboarding-list-item tip-1 flx flx-col">
-    <div class="flx flx-row al-cntr">
-      <span class="onboarding-num">1</span>
-      <span>{% ftlmsg 'onboarding-tip-1' %}</span>
-    </div>
-    <img class="onboarding-img img-arrow-up" src="{% static 'images/dashboard-onboarding/tip1-icon.svg' %}" aria-hidden="true" alt="">
-  </li>
-  <li class="dashboard-onboarding-list-item tip-2 flx flx-col">
-    <div class="flx flx-row al-cntr">
-      <span class="onboarding-num">2</span>
-      <span>{% ftlmsg 'onboarding-tip-2' %}</span>
-    </div>
-    <img class="onboarding-img img-input-icon" src="{% static 'images/dashboard-onboarding/input-icon.svg' %}" alt="Where possible, a button to create a new alias will appear inside a form's email input.">
-  </li>
-  <li class="dashboard-onboarding-list-item tip-3 flx flx-col">
-    <div class="flx flx-row al-cntr">
-      <span class="onboarding-num">3</span>
-      <span>{% ftlmsg 'onboarding-tip-3' %}</span>
-    </div>
-    <img class="onboarding-img img-context-menu" src="{% static 'images/dashboard-onboarding/context-menu.svg' %}" aria-hidden="true">
-  </li>
-</ol>
+<section class="c-onboarding">
+  <h2>{% ftlmsg 'onboarding-headline' %}</h2>
+  <ul class="c-onboarding-steps mzp-l-columns mzp-t-columns-three">
+    <li class="c-onboarding-step">
+      <div class="c-onboarding-step-number">1</div>
```

Comment:
@peiying2 Can we hard code numbers like this, or is there a better method?

Reply:
@craigcook, you are the best person to answer this.

Reply:
I can't think of any language that would need to change this from an Arabic numeral to something else, so it seems like hard-coding is fine when it's just a number by itself.

That said, these numbers seem essentially decorative so they could also be added with CSS pseudo-elements instead of in the HTML.

Reply:
Good points. I checked a few languages, Arabic, Farsi, Bengali, to name a few, do have different ways to show numbers: https://www-dev.allizom.org/**ar**/about/manifesto/ (fa, bn)